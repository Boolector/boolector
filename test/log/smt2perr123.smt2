(assert (let (($1 (_ bv1 1))) (let (($1 (_ bv1 1))))))
