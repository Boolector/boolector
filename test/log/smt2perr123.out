log/smt2perr123.smt2:1:52: body to 'let' at line 1 column 32 missing
